tests/api.rs

```diff
@@ -19,8 +19,6 @@ use rustls::TLSError;
 use rustls::sign;
 use rustls::{ALL_CIPHERSUITES, SupportedCipherSuite};
 use rustls::KeyLog;
-#[cfg(feature = "quic")]
-use rustls::quic::{self, QuicExt, ClientQuicExt, ServerQuicExt};
```

Comment:
Somebody needs to update the QUIC tests for this test. I just commented them out as I don't have the time or patience for it. Somebody will have to take this over from me.

Reply:
See https://github.com/briansmith/rustls/pull/1

Reply:
> See [briansmith#1](https://github.com/briansmith/rustls/pull/1)

Awesome! It seems reasonable to me and I merged into into this PR.